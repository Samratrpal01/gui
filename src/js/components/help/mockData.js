export const helpProps = {
  docsVersion: undefined,
  isHosted: false,
  menderVersion: 'master',
  isEnterprise: true,
  tenantCapabilities: {},
  tokens: [],
  versions: {
    repos: {
      mender: '3.3.0',
      'mender-artifact': '3.8.0',
      'mender-binary-delta': '1.3.1',
      'mender-cli': '1.8.0',
      'mender-configure-module': '1.0.4',
      'mender-connect': '2.0.2',
      'mender-convert': '3.0.0',
      'mender-gateway': '1.0.0',
      'monitor-client': '1.2.0'
    },
    releaseDate: '01-01-2003'
  }
};
