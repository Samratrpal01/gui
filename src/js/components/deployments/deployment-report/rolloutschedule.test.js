import React from 'react';

import { defaultState, undefineds } from '../../../../../tests/mockData';
import { render } from '../../../../../tests/setupTests';
import DeploymentStatus from './deploymentstatus';

describe('DeploymentStatus Component', () => {
  it('renders correctly', async () => {
    const { baseElement } = render(<DeploymentStatus deployment={defaultState.deployments.byId.d2} />);
    const view = baseElement.firstChild;
    expect(view).toMatchSnapshot();
    expect(view).toEqual(expect.not.stringMatching(undefineds));
  });
});
