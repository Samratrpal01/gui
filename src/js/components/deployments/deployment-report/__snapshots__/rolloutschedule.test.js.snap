// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeploymentStatus Component renders correctly 1`] = `
<div>
  <div
    class="progressStatus flexbox space-between centered margin-bottom "
  >
    <div
      class="flexbox column"
    >
      <div
        class="muted"
      >
        Status
      </div>
      <h4
        class="margin-bottom-none muted"
      >
        Pending
         (awaiting devices)
      </h4>
    </div>
    <div
      class="flexbox space-between align-right"
      style="min-width: 40%;"
    >
      <div
        class="flexbox column"
      >
        <div
          class="muted margin-bottom-small"
        >
          # devices
        </div>
        <div>
          1
        </div>
      </div>
      <div
        class="flexbox column disabled"
      >
        <div
          class="muted margin-bottom-small"
        >
          Skipped
        </div>
        <div
          class="status"
        >
          0
        </div>
      </div>
      <div
        class="flexbox column disabled"
      >
        <div
          class="muted margin-bottom-small"
        >
          Paused
        </div>
        <div
          class="status"
        >
          0
        </div>
      </div>
      <div
        class="flexbox column "
      >
        <div
          class="muted margin-bottom-small"
        >
          Pending
        </div>
        <div
          class="status"
        >
          1
        </div>
      </div>
      <div
        class="flexbox column disabled"
      >
        <div
          class="muted margin-bottom-small"
        >
          In Progress
        </div>
        <div
          class="status"
        >
          0
        </div>
      </div>
      <div
        class="flexbox column disabled"
      >
        <div
          class="muted margin-bottom-small"
        >
          Success
        </div>
        <div
          class="status"
        >
          0
        </div>
      </div>
      <div
        class="flexbox column disabled"
      >
        <div
          class="muted margin-bottom-small"
        >
          Fail
        </div>
        <div
          class="status"
        >
          0
        </div>
      </div>
    </div>
    <div
      class="break-all two-columns column-data  "
      style="align-items: center; align-self: flex-start; grid-template-columns: max-content 1fr; max-width: 25vw;"
    >
      <div
        class="align-right  muted"
      >
        <b>
          Max attempts per device
        </b>
      </div>
      <div
        class="flexbox "
      >
        1
      </div>
      <div
        class="align-right  muted"
      >
        <b>
          Maximum number of devices
        </b>
      </div>
      <div
        class="flexbox "
      >
        N/A
      </div>
    </div>
  </div>
</div>
`;
